src/main/java/com/gargoylesoftware/htmlunit/javascript/host/file/File.java

```diff
@@ -43,15 +47,38 @@
     private static final String LAST_MODIFIED_DATE_FORMAT = "EEE MMM dd yyyy HH:mm:ss 'GMT'Z (zzzz)";
     private static final String LAST_MODIFIED_DATE_FORMAT_FF = "EEE MMM dd yyyy HH:mm:ss 'GMT'Z";
 
+    private static final String OPTIONS_TYPE_NAME = "type";
+    //default according to https://developer.mozilla.org/en-US/docs/Web/API/File/File
+    private static final String OPTIONS_TYPE_DEFAULT = "";
+    private static final String OPTIONS_LASTMODIFIED = "lastModified";
+
     private java.io.File file_;
 
-    /**
-     * Creates an instance.
-     */
-    @JsxConstructor({CHROME, FF, FF68})
+    private String name;
+
+    private String fileType;
+
+    private long lastModified;
+
+    private long length;
+
     public File() {
     }
 
+    @JsxConstructor({CHROME, FF, FF68})
+    public File(NativeArray bytes, String name, NativeObject options) {
```

Comment:
I'm not sure if there is a more appropiate Class for the potential options here. Optional didn't work.